make_rpm

```diff
@@ -124,15 +124,15 @@ if [[ $skip_build == 0 ]]; then
 
 	# Enables/verifies the environment for the build
 	source scl_source enable ${rhpg_pkg}
-	status_rh_postgresql96=$?
+	status_rh_postgresql13=$?
 
 	source scl_source enable ${rhgcc_pkg}
 	status_devtoolset=$?
 
 	command -v pg_isready > /dev/null
 	status_pg_isready=$?
 
-	if [[ $status_rh_postgresql96 != 0 ]]; then
+	if [[ $status_rh_postgresql13 != 0 ]]; then
```

Comment:
Please rename `status_rh_postgresql13` to `status_rh_postgresql`, so that it becomes generic.

Reply:
fixed